.github/workflows/README.md

```diff
@@ -214,6 +215,92 @@ jobs:
       github-token: "${{ secrets.GH_TOKEN }}"
 ```
 
+---
+
+## Release Tag Versions
+
+This method enables the release of tag versions as well as the creation of a new snapshot version for developers to work on the next release. The workflow allows you to choose the sort of release that will be performed as well as how to generate the snapshot version.
+
+### Prerequisites
+
+Your Helm chart and `.bumpversion.cfg` need to be located inside the `charts-dir` folder of your repository (repository root by default) to use this workflow. A minimal configuration with `charts-dir=charts` could look like this:
```

Comment:
what does this workflow have to do with Helm charts? Shouldn't it be generic for any kind of project that uses bump2version?